### ISSUE TITLE
Create table like existing not supported

### ISSUE THREAD

**Issue by tharwan** [tekumara/fakesnow#39]
Snowflake supports creating a table based on an existing table via LIKE
```SQL
CREATE OR REPLACE TEMP TABLE new_table LIKE existing_table
```

Since this is not supported in duckdb, sqlglot does not transpile it. Maybe fakesnow should at least create better error messages in such cases?

**Comment by tharwan** [comment]
While looking for workarounds I realised that can be done in duckdb via the create table syntax:
```sql
CREATE TABLE t1 AS FROM t2 LIMIT 0;
```

But then again I am not sure if this would be better placed as an sqlglot issue?

**Comment by tharwan** [comment]
fixed in https://github.com/tobymao/sqlglot/issues/2922

would it be an option to not pin the version of sqlglot?

**Comment by tekumara** [comment]
Great to see this fixed in sqlglot.

sqlglot major versions aren't backwards compatible and can and have broken fakesnow in the past ([example](https://github.com/tekumara/fakesnow/commit/9177a7f1f2a20c87da204bb5278cc5a909022ec6)). So I keep it pinned to ensure fakesnow is compatible.

**Comment by tekumara** [comment]
sqlglot was bumped to 21.0.1 in e7c6c6e2b8571b047a225810571661179c5a3ec7 and released by #36
